serverless.component.yml

```diff
@@ -235,3 +235,15 @@ actions:
                     type: string
   remove:
     definition: Remove your Express.js application
+  list_alias:
+    definition: Show your aliases
```

Comment:
If I remember it right, in yesterday's product meeting, we have decided to use Chinese for all the descriptions in serverless.component.yml and have a plan to auto-generate configuration.md according to this file.

cc @tinafangkunding @AprilJC @yugasun

Reply:
If use Chinese to describe the COMMANDs,i need the  descriptions from @tinafangkunding